src/grt/src/fastroute/src/graph2d.cpp

```diff
@@ -241,6 +338,63 @@ void Graph2D::addUsageH(const int x, const int y, const int used)
   }
 }
 
+// Updates the list of congested nets.
+void Graph2D::updateCongList(const std::string& net_name,
+                             const double edge_cost)
+{
+  if (edge_cost > 0) {
+    congestion_nets_.insert(net_name);
+  } else {
+    congestion_nets_.erase(net_name);
+  }
+}
+
+// Prints all congested nets.
+void Graph2D::printAllElements()
+{
+  if (congestion_nets_.empty()) {
+    std::cout << "No congestion nets.\n";
+    return;
+  }
+
+  std::cout << "Congestion Nets: ";
+  std::cout << congestion_nets_.size();
+  std::cout << " { ";
+  for (auto it = congestion_nets_.begin(); it != congestion_nets_.end(); ++it) {
+    if (it != congestion_nets_.begin()) {
+      std::cout << ", ";
+    }
+    std::cout << "\"" << *it << "\"";
+  }
+  std::cout << " }\n";
+}
```

Comment:
I believe it is only used for debug. But even so, change the `std::cout` to use `Logger::report` or `Logger::debugPrint`